zapier/introduction/zaps-ii/zapier-search-and-create.md

```diff
@@ -0,0 +1,36 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Search and Create
+
+---
+## Content
+
+Depending on the app that you are using, Zapier allows you to create date after doing a search action.
+
+A good workflow example would be to monitor some data, and when a certain condition is met you create a new record/data.
+
+Let's take another look at the zap we have created. After you receive a message that a person was given access to your base, you want to write a new cell that specifies the number of access days.
+
+Go ahead and add another Google Sheets step to our zap, right at the end of the flow. This time, we want to create a new column, so you would have to search the `'Choose Action Event'` drop-down for the `'Create Spreadsheet Column'` option.
+
+![search-and-create](https://img.enkipro.com/4da04371ac9c9efc7769f35c39b275b2.png)
+
+---
+## Practice
```

Comment:
one question that came to mind could be formulated like this:

What actions are available for a Google Sheet step in a zapier workflow? If unsure, you can always check the app page -> link to page